arangod/Pregel/Conductor.cpp

```diff
@@ -480,8 +478,22 @@ ErrorCode Conductor::_initializeWorkers(std::string const& suffix,
   }
 
   _dbServers.clear();
-  for (auto const& pair : vertexMap) {
-    _dbServers.push_back(pair.first);
+  workers.clear();
```

Comment:
I don't  understand why this clearing is necessary; in the lifetime of a conductor this variable should only have the states "initialized empty" and "all the workers in there", shouldn't it?

Maybe it's better to have this be a `std::unique_ptr` to reflect this fact?

Reply:
Ah, I added it because the recovery state would also call the initializeWorkers function again. We deleted recovery, so I will delete this clearing.